hrjobcontract/js/src/job-contract/services/contract-leave.js

```diff
@@ -194,10 +210,15 @@ define([
               return this.leave_amount;
             };
 
-            angular.forEach(leaveType, function (type, typeId) {
-              modelEntry.leave_type = typeId;
+            _.each(leaveType, function (type, typeId) {
+              modelEntry.leave_type = type.id;
```

Comment:
Can we do
```
model.push({
  leave_type: type.id,
  ....
});
```

Code will be simpler in my opinion.

Reply:
modelEntry already has some props, so in that case it makes sense how it was implemented (please note, I just changed Angular.each to _.each here + added extra props).